packages/customer-effort-score/README.md

```diff
@@ -14,4 +14,66 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 ## Usage
 
-Coming soon.
+### CustomerEffortScore component
+
+`CustomerEffortScore` is a React component that can be used to implement your
+own effort score survey, providing your own logging infrastructure.
+
+This creates a wrapper component around `CustomerEffortScore` which simply logs
+responses to the console:
+
+```
```

Comment:
In Github you can make it super fancy for js. Same for the block below

```suggestion
```js
```

Reply:
I always forget this. Thanks :-)